packages/server-wallet/src/mock-chain-service/index.ts

```diff
@@ -9,9 +17,42 @@ type FundChannelArg = {
   expectedHeld: Uint256;
   amount: Uint256;
 };
-export class OnchainService {
+export type SetFundingArg = {
+  channelId: Bytes32;
+  assetHolderAddress: Address;
+  amount: Uint256;
+};
+
+export interface ChainEventListener {
+  setFunding(arg: SetFundingArg): void;
+}
+export class OnchainService implements OnchainServiceInterface {
   static fundChannel(_arg: FundChannelArg): Promise<SubmissionStatus> {
     const submissionStatus: SubmissionStatus = 'Success';
     return Promise.resolve(submissionStatus);
   }
+
+  registerChannel(_channelId: Bytes32, _assetHolders: Address[]): Promise<void> {
+    return Promise.resolve();
+  }
+
+  // TODO: remove in v1
```

Comment:
Should this be removed?

Reply:
Chain Service v1 eliminates the need for both `mock-chain-service` and `onchain-service` (which is chain service v0). My thought was to remove those in a separate PR in order to reduce the change set of this PR and make it easier to roll back changes if that needs to be done.